CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v2.0.0-rc.1 / 2021-03-26
+
+* [CHANGE] Rename --labels-metric-allow-list to --metric-labels-allowlist #1424
```

Comment:
We also want to stop creating quay images, not sure if this is the right place to mention this or in the release itself, or both.

See https://github.com/kubernetes/kube-state-metrics/issues/1409 for background.

Reply:
I added a note about quay.io and k8s.gcr.io